### DISCUSSION TITLE
BatchEasyHardMiner Docs

### DISCUSSION THREAD

**Discussion by suchith-sixsense** [KevinMusgrave/pytorch-metric-learning#607] [Q&A] [answered]
As per the docs of the [BatchEasyHardMiner](https://kevinmusgrave.github.io/pytorch-metric-learning/miners/#batcheasyhardminer), 

> To implement the loss function described in the paper, use this miner in combination with [NTXentLoss(temperature=0.1)](https://kevinmusgrave.github.io/pytorch-metric-learning/losses/#ntxentloss).

The paper that is in the discussion is [this](http://openaccess.thecvf.com/content_WACV_2020/papers/Xuan_Improved_Embeddings_with_Easy_Positive_Triplet_Mining_WACV_2020_paper.pdf)

According to the paper, one of the loss function they are using is this :
<img width="464" alt="image" src="https://user-images.githubusercontent.com/110653781/230789823-518b21f9-4f18-4e14-a44c-70acfebe92d9.png">

which implies we have to use `temperature = 1.0` and not `temperature = 0.1`. So want to know if I am missing something or it is mentioned wrongly in docs.

Thanks.

**Comment by KevinMusgrave** [comment, marked as answer]
The paper says "The loss function is based on NCA which has a single parameter, temperature, and in all cases we set this parameter to 0.1."

**Reply by suchith-sixsense** [reply]
Sorry my bad, I didn't go through section 5.1 and was assuming `temperature = 1.0` from the expression of loss function. Apologies for the same.
